pages/common/nu.md

```diff
@@ -0,0 +1,24 @@
+# nu
+
+> Nushell ("a new type of shell") takes a modern, structured approach to your command-line.
+> More information: <https://www.nushell.sh>.
+
+- Start an interactive shell session:
+
+`nu`
+
+- Execute a command and then exit:
+
+`nu --commands "{{command}}"`
```

Comment:
How can I send arguments to the target command? (check [nix-shell](https://github.com/tldr-pages/tldr/blob/master/pages/common/nix-shell.md) and [kde-inhibit](https://github.com/tldr-pages/tldr/blob/master/pages/linux/kde-inhibit.md) for examples).

Reply:
Not sure it applies to `nushell`. If I understand well (I am a `nushell` newbie 😇), `--commands`'s value is a function pipeline (function(s) with arguments or/and operators):  
```
nu --commands "echo 42"
```
```
nu --commands "ls | sort-by size | reverse"
```

Reply:
Cool, so it's like `nix-shell`. We can add this to make it clear in the example:
```suggestion
`nu --commands "{{command}} {{command_arguments}}"`
```

Reply:
Hmm, I think we cannot say that a Nushell command has optional arguments. If I understand well, commands can have parameters and can be combined with operators, but I am not sure we have to mention this in `tldr`. :nerd_face:

Reply:
Yeah, I think it's best to keep things simple here. Let's keep this as-is.